src/input/capability.rs

```diff
@@ -430,27 +430,24 @@ pub enum GamepadButton {
     RightStick,
     /// Touch binary sensor for right stick
     RightStickTouch,
+    /// Nintendo codes used on some rare devices & d_input
+    C,
+    Z,
```

Comment:
What buttons are these? What buttons would emit these codes?

The general vision I had with capabilities was that they would describe either:
1. The physical location of the input, or
2. A high-level specific function (e.g. `Screenshot`, `Guide`)

So for example, if a DInput device emits a `Z` event when the left trigger is pressed, we should just be translating `Z` events into the `LeftTrigger` capability.